src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,39 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    @include square($cell-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      transition: 1s;
```

Comment:
First - it should be half a second:
![image](https://user-images.githubusercontent.com/60885495/113708411-db8e1400-96e9-11eb-9e93-eaef89243801.png)

Second - if you write transition under `&__day` selector, not on hover, it will work both ways.

Third - always list styles that you apply transition for:
```suggestion
      transition: transform 0.5s, background-color 0.5s;
```